literatur.bib

```diff
@@ -2675,6 +2675,13 @@ @Proceedings{celorrio-et-al-2011
   year =         "2011"
 }
 
+@InProceedings{celorrio-et-al-icaps2013wspal,
+  author =       "Sergio J. Celorrio and Patrick Haslum and Sylvie Thiebaux",
+  title =        "Pruning bad quality causal links in sequential satisfying planning",
```

Comment:
Use title case (except if the title in the PDF really only uses lower case). Here and for other entries.

Reply:
Please add `% Title uses lower case.` above the entry.